tools/releaseBuild/setVstsVariables.ps1

```diff
@@ -1,5 +1,5 @@
 $vstsVariables = @{
-    PSES_BRANCH = 'master'
+    PSES_BRANCH = 'move-to-yml'
```

Comment:
Will this need changing back at some point?

Reply:
```suggestion
    PSES_BRANCH = 'master'
```